### PR TITLE
5065 replace jcenter by mavenCentral

### LINKED ISSUE #5065: Replace jcenter Maven repository with an alternative
### Feature Description

jcenter is one of the repositories we're using both for the web app (sormas-base/pom.xml) as well as in the mobile application (gradle.build).

### Problem Description

Unfortunately, the repository will shut down by the end of April. While it will remain functional until February 2022, there will be no updates or new submissions to the repository.

### Proposed Change

We need to migrate to a different Maven repository that can replace jcenter. Before that, we need to make a decision which repository we can use instead.

### Additional Information

Some resources:
https://medium.com/codechai/jcenter-bintray-shutting-down-2e029d98a812
https://developer.android.com/studio/build/jcenter-migration
https://blog.gradle.org/jcenter-shutdown

### PR DESCRIPTION
**Pull request by AxelNennker**
Signed-off-by: Axel Nennker <axel.nennker@telekom.de>

Remove jcenter()
Replace openbeans 1.0 by openbeans 1.0.2 because that is in mavenCentral()
Downgrade me.dm7.barcodescanner:zxing from 1.9.13 (jcenter) to 1.9.8 (mavenCentral)

Fixes #5065

### CODE REVIEW

File: sormas-app/app/build.gradle [hunk 1]
```diff
@@ -131,7 +130,7 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     androidTestImplementation 'androidx.test.uiautomator:uiautomator:2.2.0'
     implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0'
-    implementation 'me.dm7.barcodescanner:zxing:1.9.13'
+    implementation 'me.dm7.barcodescanner:zxing:1.9.8'
```
Comment (MateStrysewske) [inline sormas-app/app/build.gradle:133]: What's the reason for the downgrade here? Does mavenCentral not contain 1.9.13?

Comment (AxelNennker) [reply sormas-app/app/build.gradle:133]: Exactly. 1.9.13 is in jcenter, but maven central has 1.9.8

**Comment by JonasCir** [comment]
@MateStrysewske I wanted to take care of #7513 (which seems to be a duplicate of #5065) and I stumbled over this PR. Changes seem fine to me and I don't want to do duplicate work, so is there a chance to reopen the PR?

@AxelNennker thanks for taking care of this!

:christmas_tree:

**Comment by MateStrysewske** [comment]
@JonasCir I believe I closed this PR on accident when we did the Android dependency upgrades. Didn't want to merge it back then because I didn't want to have so many changes at once. Unfortunately the branch is deleted now, but we can replicate the changes anyway.

**Pull request closed without merging**